Prose/ProseLib/Sources/SidebarFeature/SidebarReducer.swift

```diff
@@ -0,0 +1,73 @@
+import ComposableArchitecture
+import Foundation
+import SharedModels
+
+public typealias UserCredentials = JID
```

Comment:
As said just before, maybe we can rename the type right now, just so we don't keep around a name closely related to `AuthenticationFeature`'s `UserCredentials`.

WDYT?

Reply:
See above.